Dockerfile

```diff
@@ -10,16 +10,33 @@ LABEL name="custodian" \
 # to minimize size
 ADD setup.py README.md requirements.txt /src/
 ADD c7n /src/c7n/
-ADD tools /src/tools/
+ADD tools/c7n_gcp /src/tools/c7n_gcp
+ADD tools/c7n_azure /src/tools/c7n_azure
+ADD tools/c7n_mailer /src/tools/c7n_mailer
```

Comment:
? we already have a separate mailer docker image, why install it here?

Reply:
nevermind, just realized its to slim size by picking individual packages of interest. still curious why the mailer is here if we have a separate docker image for it. c7n cli wouldn't be deploying mailer.

Reply:
Yeah I didn't realize we had a separate one and was going to start leveraging it here, will clean that up, should bring down the size a decent bit too.

Reply:
I am all for having a consolidated image with both mailer AND custodian installed. We are leveraging AWS CodeBuild to execute C7N and having a single docker image could significantly speed up our process

Reply:
@kapilt and @stefangordon  can we add back mailer please?

Reply:
If we did keep it, using it directly is a little harder since you also have to include `--entrypoint "/usr/local/bin/mailer"`, I was going to have our cask docker wrapper do this for you if you ran `custodian-cask mailer [whatever]`  - but that might create confusion too.